public/ledger.js

```diff
@@ -383,28 +383,27 @@ class Ledger {
   /**
    * Send tokens and signatures to ledger before a tx
    *
-   * @param {Object} tokens List of data to send to ledger
-   *   List of data with the format:
+   * @param {Object} tokenMap of data to send to Ledger
+   *   keys are the token uid and values are the token info in the format:
```

Comment:
Should we write that token info is '...token info serialized as buffer in the format:'? 

(both comments in this method are applied to the method below as well

Reply:
Done in [86d6ee0](https://github.com/HathorNetwork/hathor-wallet/pull/264/commits/86d6ee02563d136387db484d7848ea2da6a5a3a4)